### PR TITLE
Add fields to ig (sync with public api)

### PR DESCRIPTION
**Pull request by GirevoyT**
Add status, instance_template.name, instance_template.hostname, instances.status_changed_at fields to instance groups.

### CODE REVIEW

File: website/docs/d/datasource_compute_instance_group.html.markdown [hunk 1]
```diff
@@ -163,6 +165,8 @@ The structure is documented below.
 The structure is documented below.
 * `network_settings` -  Network acceleration settings.
 The structure is documented below.
+* `name` - Name template of the instance.
+* `hostname` - Host name temaplate for the instance.
```
Comment (GennadySpb) [inline website/docs/d/datasource_compute_instance_group.html.markdown]:
```suggestion
* `hostname` - Hostname template for the instance.
```

File: website/docs/r/compute_instance_group.html.markdown [hunk 2]
```diff
@@ -186,6 +184,26 @@ The structure is documented below.
...
 
+* `name` - (Optional) Name of the instance.  
+In order to be unique it must contain at least on of instance unique placeholders:   
+{instance.short_id}   
+{instance.index}   
+combination of {instance.zone_id} and {instance.index_in_zone}   
+Example: my-instance-{instance.index}  
+If not set, default is used: {instance_group.id}-{instance.short_id}   
+It may also contain another placeholders, see metadata doc for full list.
+* `hostname` - (Optional) Host name for the instance.
```
Comment (GennadySpb) [inline website/docs/r/compute_instance_group.html.markdown]:
```suggestion
* `hostname` - (Optional) Hostname template for the instance.
```

File: website/docs/r/compute_instance_group.html.markdown [hunk 3]
```diff
@@ -186,6 +184,26 @@ The structure is documented below.
 
 * `network_settings` - (Optional) Network acceleration type for instance. The structure is documented below.
 
+* `name` - (Optional) Name of the instance.  
+In order to be unique it must contain at least on of instance unique placeholders:
```
Comment (GennadySpb) [inline website/docs/r/compute_instance_group.html.markdown]:
```suggestion
In order to be unique it must contain at least one of instance unique placeholders:   
```

File: website/docs/r/compute_instance_group.html.markdown [hunk 4]
```diff
@@ -186,6 +184,26 @@ The structure is documented below.
...
+The host name must be unique within the network and region.   
+If not specified, the host name will be equal to [yandex.cloud.compute.v1.Instance.id] of the instance   
+and FQDN will be `<id>.auto.internal`. Otherwise FQDN will be `<hostname>.<region_id>.internal`.   
+In order to be unique it must contain at least on of instance unique placeholders:   
+ {instance.short_id}   
+ {instance.index}   
+ combination of {instance.zone_id} and {instance.index_in_zone}   
+Example: my-instance-{instance.index}   
+If not set, `name` value will be used   
+It may also contain another placeholders, see metadata doc for full list.
```
Comment (GennadySpb) [inline website/docs/r/compute_instance_group.html.markdown:206]: Please set link for metadata doc.

File: website/docs/r/compute_instance_group.html.markdown [hunk 5]
```diff
@@ -186,6 +184,26 @@ The structure is documented below.
 
 * `network_settings` - (Optional) Network acceleration type for instance. The structure is documented below.
 
+* `name` - (Optional) Name of the instance.
```
Comment (GennadySpb) [inline website/docs/r/compute_instance_group.html.markdown]:
```suggestion
* `name` - (Optional) Name template of the instance.
```
?

File: website/docs/r/compute_instance_group.html.markdown [hunk 6]
```diff
@@ -186,6 +184,26 @@ The structure is documented below.
...
+* `name` - (Optional) Name of the instance.  
+In order to be unique it must contain at least on of instance unique placeholders:   
+{instance.short_id}   
+{instance.index}   
+combination of {instance.zone_id} and {instance.index_in_zone}   
+Example: my-instance-{instance.index}  
+If not set, default is used: {instance_group.id}-{instance.short_id}   
+It may also contain another placeholders, see metadata doc for full list.
+* `hostname` - (Optional) Host name for the instance.   
+This field is used to generate the [yandex.cloud.compute.v1.Instance.fqdn] value.
```
Comment (GennadySpb) [inline website/docs/r/compute_instance_group.html.markdown]:
```suggestion
This field is used to generate the FQDN value of instance.
```
?

File: website/docs/r/compute_instance_group.html.markdown [hunk 7]
```diff
@@ -186,6 +184,26 @@ The structure is documented below.
...
+In order to be unique it must contain at least on of instance unique placeholders:   
+{instance.short_id}   
+{instance.index}   
+combination of {instance.zone_id} and {instance.index_in_zone}   
+Example: my-instance-{instance.index}  
+If not set, default is used: {instance_group.id}-{instance.short_id}   
+It may also contain another placeholders, see metadata doc for full list.
+* `hostname` - (Optional) Host name for the instance.   
+This field is used to generate the [yandex.cloud.compute.v1.Instance.fqdn] value.   
+The host name must be unique within the network and region.
```
Comment (GennadySpb) [inline website/docs/r/compute_instance_group.html.markdown]:
```suggestion
The hostname must be unique within the network and region.   
```

File: website/docs/r/compute_instance_group.html.markdown [hunk 8]
```diff
@@ -186,6 +184,26 @@ The structure is documented below.
...
+{instance.short_id}   
+{instance.index}   
+combination of {instance.zone_id} and {instance.index_in_zone}   
+Example: my-instance-{instance.index}  
+If not set, default is used: {instance_group.id}-{instance.short_id}   
+It may also contain another placeholders, see metadata doc for full list.
+* `hostname` - (Optional) Host name for the instance.   
+This field is used to generate the [yandex.cloud.compute.v1.Instance.fqdn] value.   
+The host name must be unique within the network and region.   
+If not specified, the host name will be equal to [yandex.cloud.compute.v1.Instance.id] of the instance
```
Comment (GennadySpb) [inline website/docs/r/compute_instance_group.html.markdown]:
```suggestion
If not specified, the hostname will be equal to id of the instance   
```

File: yandex/structures.go [hunk 9]
```diff
@@ -1299,6 +1311,12 @@ func flattenInstances(instances []*instancegroup.ManagedInstance) ([]map[string]
 		instDict["status_message"] = instance.GetStatusMessage()
 		instDict["zone_id"] = instance.GetZoneId()
 
+		changeAt, err := getTimestamp(instance.GetStatusChangedAt())
+		if err != nil {
+			return res, err
+		}
+		instDict["status_changed_at"] = changeAt
```
Comment (GennadySpb) [inline yandex/structures.go]:
```suggestion
		changedAt, err := getTimestamp(instance.GetStatusChangedAt())
		if err != nil {
			return res, err
		}
		instDict["status_changed_at"] = changedAt
```

File: website/docs/d/datasource_compute_instance_group.html.markdown [hunk 10]
```diff
@@ -228,6 +232,7 @@ The `instances` block supports:
 * `zone_id` - The ID of the availability zone where the instance resides.
 * `network_interface` - An array with the network interfaces attached to the managed instance.
 The structure is documented below.
+* `status_changed_at` -The timestamp  when the status of the managed instance was last changed.
```
Comment (GennadySpb) [inline website/docs/d/datasource_compute_instance_group.html.markdown]:
```suggestion
* `status_changed_at` -The timestamp when the status of the managed instance was last changed.
```

**Review by GennadySpb — APPROVED** [review]
LGTM.

File: website/docs/r/compute_instance_group.html.markdown (hunk 4 above)
Comment (GirevoyT) [reply website/docs/r/compute_instance_group.html.markdown:206]: No documentetion :(

**Pull request merged**